addons/mass_mailing/views/res_config_settings_views.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+.<?xml version="1.0" encoding="utf-8"?>
```

Comment:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```

Reply:
Thanks for pointing this one out, saved me searching for it. I've resolved it :)